### PR TITLE
SPCTR-385: Fix- Testimonial block arrow and content margin issue

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

The testimonials arrow does not look good on a mobile devices.

https://brainstormforce.slack.com/archives/CAQ2TSA6A/p1638382266070200 - Connect your Slack account 

Screenshot - [Image 2021-12-02 at 10.23.54 AM](https://share.getcloudapp.com/BluD6Q5w)

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -166,6 +166,9 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 == Changelog ==
 
+= x.x.x =
+* Fix: Testimonial - Resolved testimonial arrow and content margin issue.
```
Comment (bsf-zanev) [inline readme.txt:170]:
```suggestion
* Fix: Testimonial - Resolved an inssue in which the arrows would overlap the content on mobile devices.
```

**Review by bsf-zanev — APPROVED** [review]

**Review by bsf-zanev — APPROVED** [review]

File: src/blocks/testimonial/style.scss [hunk 2]
```diff
@@ -104,6 +104,11 @@
 	visibility: hidden;
 }
 
+.uagb-tm__desc {
```
Comment (AbhijitNage123) [inline src/blocks/testimonial/style.scss:107]: CSS will be apply on desktop and tablet and testimonial arrow issue is on Mobile device. Can we write mobile specific CSS? @rajmohanreddy774

File: readme.txt (hunk 1 above)
Comment (rajmohanreddy774) [reply readme.txt]: updated

**Review by AbhijitNage123 — APPROVED** [review]

File: src/blocks/testimonial/style.scss [hunk 3]
```diff
@@ -298,4 +298,9 @@
 			}
 		}
 	}
+
+	.uagb-tm__desc {
```
Comment (aapatre) [inline src/blocks/testimonial/style.scss:302]: Since this is a mobile only fix, do we need to use the `media` query?

Comment (aapatre) [reply src/blocks/testimonial/style.scss:302]: Approved as discussed (mobile only styling is being applied via a special class)

**Review by adi3890 — COMMENTED** [review]
Please check for merge conflict.

**Review by adi3890 — APPROVED** [review]

**Pull request merged**